fiftyone/__public__.py

```diff
@@ -5,13 +5,11 @@
 | `voxel51.com <https://voxel51.com/>`_
 |
 """
-import fiftyone.core.config as foc
-import fiftyone.core.odm as foo
+import fiftyone.core.config as _foc
```

Comment:
`_foc` is just to avoid polluting the `fiftyone` package namespace (should have done it earlier)